### PR TITLE
List Liquid Bitcoin (L-BTC)

### PR DESCRIPTION
**Pull request by wiz**
### Asset name

Liquid Bitcoin

### Ticker

L-BTC

### Website

https://liquid.net

### Block explorer

https://blockstream.info/liquid/

### Additional technical requirements

Due to base layer privacy aka Confidential Transactions, the Bisq mediators / refund agents must obtain the Blinding Key from the L-BTC sender to verify transactions for any trade dispute, similar to Monero. 

While the [full documentation](https://docs.blockstream.com/liquid/developer-guide/confidential-transactions.html?highlight=blinding%20key) explains all the details, I will make a separate PR to docs repo for use by support agents, mediators, and refund agents with simple steps to verify trade funds were sent.

### CODE REVIEW

**Comment by ripcurlx** [comment]
> While the [full documentation](https://docs.blockstream.com/liquid/developer-guide/confidential-transactions.html?highlight=blinding%20key) explains all the details, I will make a separate PR to docs repo for use by support agents, mediators, and refund agents with simple steps to verify trade funds were sent.

@wiz I think it would be good to have this already before merging as we need to check the efforts with mediators and refund agents if it is ok with them as well. Just in case we get lots of mediation/refund agent cases this might cause lots of workload for verifying transactions.

**Comment by wiz** [comment]
@ripcurlx When we listed other crypto assets with base layer privacy like Monero, Zcash, etc. did you require a separate bisq-docs contribution to document verification steps for mediators, or did they simply refer to the generic documentation for verification? If the former, could you point me to where this is currently documented so I can use that as a reference example for my bisq-docs PR?

**Comment by m52go** [comment]
Since these docs are probably a better fit for the knowledge base, and since the knowledge base should be going live in the next 1-2 weeks (before v1.2.6 release), I think it's better if we wait to specify these verification steps there. 

I don't think there's an obvious place for them in the current documentation repository.

**Comment by wiz** [comment]
After further discussion with @ripcurlx, I added the following popup dialog when adding an L-BTC payment account

Screen Shot 2020-01-22 at 4 03 55

**Comment by wiz** [comment]
After further discussion with @ripcurlx, I added the following popup dialog message that will be displayed to users when configuring their L-BTC payment account

Screen Shot 2020-01-23 at 1 42 30

**Comment by wiz** [comment]
After further discussion with @ripcurlx, I added the following popup dialog message that will be displayed to users when adding the L-BTC payment account

Screen Shot 2020-01-23 at 4 18 43

**Comment by ripcurlx** [comment]
I'll merge this PR now after I've talked to mediators and refund agent about payment verification procedure.

**Review by ripcurlx — APPROVED** [review]
utACK

**Comment by wiz** [comment]
test

**Comment by ripcurlx** [comment]
~As I haven't ACKed asset PRs before I completely missed that the test for the asset was missing. So I wanted to create one quickly myself but I'm not able to validate addresses with the current validator implementation (e.g. [da369298d9bb312b636abd260ec1df31bfc513b2b059500879b3e95cad3b7519](https://blockstream.info/liquid/tx/da369298d9bb312b636abd260ec1df31bfc513b2b059500879b3e95cad3b7519?output:1))  should be a valid address. But when testing it the validator rejects this address. Do I understand something wrong how to get a valid L-BTC address?~ I was not looking properly and copied the tx id instead of the address. Still we found out that we only support non native segwit addresses, why we changed the error message.

**Comment by wiz** [comment]
We can revise the popup message after https://github.com/mempool/mempool/issues/523 is implemented

**Pull request merged**
